units/crons.go

```diff
@@ -1030,6 +1030,23 @@ func PopulateVolumeExpirationJob() amboy.QueueOperation {
 	}
 }
 
+func PopulateUnstickVolumesJob() amboy.QueueOperation {
+	return func(ctx context.Context, queue amboy.Queue) error {
+		catcher := grip.NewBasicCatcher()
+		volumes, err := host.FindVolumesWithTerminatedHost()
+		if err != nil {
+			return errors.Wrap(err, "finding volumes to delete")
+
+		}
+		for _, v := range volumes {
+			ts := utility.RoundPartOfHour(0).Format(TSFormat)
```

Comment:
I wonder if this should maybe be done daily instead of hourly?

Reply:
I thought about this but I'd say it's as unlikely to occur as some other hourly jobs, and the query isn't very expensive so I don't think it's worth adding a new kind of daily job.

Reply:
It looks like we have one here: https://github.com/evergreen-ci/evergreen/blob/main/units/crons.go#L948

Reply:
oh haha that's funny bc it's in the hourly list. I'd rather make a split ticket to move both this job and the stats job to an actual daily cron; it's really not expensive enough that hourly is going to be problematic as it is.

Reply:
https://jira.mongodb.org/browse/EVG-20611

Reply:
I agree. Thank you for creating the ticket!